src/mthrowu.c

```diff
@@ -360,7 +360,7 @@ ohitmon(
         /* probably thrown by a monster rather than 'other', but the
            distinction only matters when hitting the hero */
         potionhit(mtmp, otmp, POTHIT_OTHER_THROW);
-        wakeup(mtmp, FALSE);
+        wakeup(mtmp, FALSE, FALSE);
```

Comment:
This has a seemimic() right next to it; is there some reason it should stay there and this shouldn't just call wakeup with TRUE?